hack/dep-prune.sh

```diff
@@ -1,2 +1,5 @@
 #!/bin/bash
 for file in $(find vendor/ -name "*_test.go"); do rm ${file}; done
+rm -rf vendor/github.com/golang/glog
+mkdir -p vendor/github.com/golang/
+ln -s ../../../pkg/staging/glog/ vendor/github.com/golang/glog
```

Comment:
Mh, interesting idea, using a symlink in the vendor dir. But it's also a bit hacky, isn't it? :)
What do you think about this:

- put the glog impl into its own github repository
- configure glog in glide.yaml with original `package` name, but our `repo` name

I didn't test this, but I think it should work

Reply:
It would. However I would prefer if we keep it together. Note that k8s does the same. For instance `apimachinery` is symlinked into the `vendor` folder. There exists a repo for it too, but it is synced out of `kubevirt/kubevirt` and not used by it. I configured `glide` to ignored `glog`. I can actually remove the whole extra logic.

Reply:
oh, I knew k8s does sth special with apimachinery etc., but wasn't aware that it's exactly this. That also explains the `staging` directory name you use. In that case I'm fine with it.
But would you mind adding a readme to the staging dir, which explains what's happening here (a short version of https://github.com/kubernetes/kubernetes/blob/master/staging/README.md)?

Reply:
Ok, can't remove it. Since glide does not track the package, it would delete the  symlink.

Reply:
> But would you mind adding a readme to the staging dir, which explains what's happening here (a short version of

Done